internal/config/validator.go

```diff
@@ -37,6 +37,24 @@ func (v *Validator) validateCli() bool {
 		valid = false
 	}
 
+	// kubectl is always required (core Kubernetes functionality)
+	if !v.isCliInstalled("kubectl") {
+		v.errors = append(v.errors, "kubectl is not installed or not found in PATH")
+		valid = false
+	}
+
+	// helm is required for Inspektor Gadget component and when explicitly enabled
+	if !v.isCliInstalled("helm") {
```

Comment:
or should we just disable all helm related IG operations when helm is not set via additional tools?

Reply:
users might be not aware of that IG is disabled.

@mqasimsarfraz what is your suggestion for this validation?

Reply:
That is a good idea to use helm go sdk for IG handlers, thank you @mqasimsarfraz . Will change the code to not care about IG for helm cli.